### ISSUE TITLE
最新版的 API 接口调用报错

### ISSUE THREAD

**Issue by derekchen070** [RVC-Boss/GPT-SoVITS#2104]
当前使用g2pw进行拼音推理
2025-02-24 08:38:04.381012919 [E:onnxruntime:Default, provider_bridge_ort.cc:1744 TryGetProviderInfo_CUDA] /onnxruntime_src/onnxruntime/core/session/provider_bridge_ort.cc:1426 onnxruntime::Provider& onnxruntime::ProviderLibrary::Get() [ONNXRuntimeError] : 1 : FAIL : Failed to load library libonnxruntime_providers_cuda.so with error: libcublasLt.so.11: cannot open shared object file: No such file or directory

Building prefix dict from the default dictionary ...
DEBUG:jieba_fast:Building prefix dict from the default dictionary ...
Loading model from cache /tmp/jieba.cache
DEBUG:jieba_fast:Loading model from cache /tmp/jieba.cache
Loading model cost 0.658 seconds.
DEBUG:jieba_fast:Loading model cost 0.658 seconds.
Prefix dict has been built succesfully.
DEBUG:jieba_fast:Prefix dict has been built succesfully.

一直用 CPU推理，不用 GPU。搞了半天没搞定，哪位大神能帮忙看一下吗？！

**Comment by derekchen070** [comment]
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2023 NVIDIA Corporation
Built on Tue_Jun_13_19:16:58_PDT_2023
Cuda compilation tools, release 12.2, V12.2.91
Build cuda_12.2.r12.2/compiler.32965470_0



nvidia-cudnn - NVIDIA CUDA Deep Neural Network library (install script)
libcudnn8 - cuDNN runtime libraries
libcudnn8-dev - cuDNN development libraries and headers
libcudnn8-samples - cuDNN samples
cudnn - NVIDIA CUDA Deep Neural Network library (cuDNN)
cudnn9 - NVIDIA CUDA Deep Neural Network library (cuDNN)
cudnn9-cuda-11-8 - NVIDIA cuDNN for CUDA 11.8
cudnn9-cuda-11 - NVIDIA cuDNN for CUDA 11
cudnn9-cuda-12-3 - NVIDIA cuDNN for CUDA 12.3
cudnn9-cuda-12-4 - NVIDIA cuDNN for CUDA 12.4
cudnn9-cuda-12-5 - NVIDIA cuDNN for CUDA 12.5
cudnn9-cuda-12 - NVIDIA cuDNN for CUDA 12
libcudnn9-cuda-11 - cuDNN runtime libraries for CUDA 11.8
libcudnn9-dev-cuda-11 - cuDNN development headers and symlinks for CUDA 11.8
libcudnn9-samples - cuDNN samples
libcudnn9-static-cuda-11 - cuDNN static libraries for CUDA 11.8
cudnn9-cuda-12-6 - NVIDIA cuDNN for CUDA 12.6
cudnn9-cuda-12-8 - NVIDIA cuDNN for CUDA 12.8
libcudnn9-cuda-12 - cuDNN runtime libraries for CUDA 12.8
libcudnn9-dev-cuda-12 - cuDNN development headers and symlinks for CUDA 12.8
libcudnn9-static-cuda-12 - cuDNN static libraries for CUDA 12.8

**Comment by derekchen070** [comment]
absl-py==2.1.0
accelerate==1.4.0
addict==2.4.0
aiofiles==23.2.1
aiohttp==3.9.1
aiosignal==1.3.1
aliyun-python-sdk-core==2.14.0
aliyun-python-sdk-kms==2.16.2
altair==5.2.0
annotated-types==0.6.0
antlr4-python3-runtime==4.9.3
anyio==4.2.0
async-timeout==4.0.3
attrs==23.2.0
audioread==3.0.1
av==14.1.0
budoux==0.6.4
cachetools==5.3.2
certifi==2023.11.17
cffi==1.16.0
chardet==5.2.0
charset-normalizer==3.3.2
click==8.1.7
cn2an==0.5.22
coloredlogs==15.0.1
colorlog==6.9.0
contourpy==1.2.0
crcmod==1.7
cryptography==42.0.0
ctranslate2==4.5.0
cycler==0.12.1
Cython==0.29.37
datasets==2.16.1
decorator==5.1.1
dill==0.3.7
Distance==0.1.3
editdistance==0.6.2
einops==0.8.1
einx==0.3.0
exceptiongroup==1.2.0
fast-langdetect==0.2.5
fastapi==0.109.0
faster-whisper==1.1.1
fasttext-predict==0.9.2.4
ffmpeg-python==0.2.0
ffmpy==0.3.1
filelock==3.13.1
flatbuffers==23.5.26
fonttools==4.47.2
frozendict==2.4.6
frozenlist==1.4.1
fsspec==2023.10.0
funasr==1.0.27
future==0.18.3
g2p-en==2.1.0
g2pk2==0.0.3
gast==0.5.4
google-auth==2.26.2
google-auth-oauthlib==1.2.0
gradio==4.24.0
gradio_client==0.14.0
grpcio==1.60.0
h11==0.14.0
hdbscan==0.8.33
httpcore==1.0.2
httpx==0.26.0
huggingface-hub==0.29.1
humanfriendly==10.0
hydra-core==1.3.2
idna==3.6
importlib-metadata==7.0.1
importlib-resources==6.1.1
inflect==7.0.0
jaconv==0.3.4
jamo==0.4.1
jieba==0.42.1
jieba-fast==0.53
Jinja2==3.1.3
jmespath==0.10.0
joblib==1.3.2
jsonschema==4.21.1
jsonschema-specifications==2023.12.1
kaldiio==2.18.0
kiwisolver==1.4.5
ko-pron==1.3
LangSegment==0.3.5
librosa==0.9.2
lightning-utilities==0.10.1
linkify-it-py==2.0.2
llvmlite==0.39.1
loguru==0.7.3
Markdown==3.5.2
markdown-it-py==2.2.0
MarkupSafe==2.1.4
matplotlib==3.8.2
mdit-py-plugins==0.3.3
mdurl==0.1.2
modelscope==1.10.0
more-itertools==10.6.0
mpmath==1.3.0
multidict==6.0.4
multiprocess==0.70.15
networkx==3.2.1
nltk==3.8.1
numba==0.56.4
numpy==1.23.4
nvidia-cublas-cu12==12.1.3.1
nvidia-cuda-cupti-cu12==12.1.105
nvidia-cuda-nvrtc-cu12==12.1.105
nvidia-cuda-runtime-cu12==12.1.105
nvidia-cudnn-cu12==8.9.2.26
nvidia-cufft-cu12==11.0.2.54
nvidia-curand-cu12==10.3.2.106
nvidia-cusolver-cu12==11.4.5.107
nvidia-cusparse-cu12==12.1.0.106
nvidia-nccl-cu12==2.18.1
nvidia-nvjitlink-cu12==12.3.101
nvidia-nvtx-cu12==12.1.105
oauthlib==3.2.2
omegaconf==2.3.0
onnxruntime==1.16.3
onnxruntime-gpu==1.18.1
openai-whisper==20240930
OpenCC==1.1.1
orjson==3.9.12
oss2==2.18.4
packaging==23.2
pandas==2.2.0
peft==0.14.0
pillow==10.2.0
pip==23.3.1
platformdirs==4.1.0
pooch==1.8.0
proces==0.1.7
protobuf==4.23.4
psutil==5.9.8
py3langid==0.2.2
pyarrow==15.0.0
pyarrow-hotfix==0.6
pyasn1==0.5.1
pyasn1-modules==0.3.0
pycparser==2.21
pycryptodome==3.20.0
pydantic==2.5.3
pydantic_core==2.14.6
pydub==0.25.1
Pygments==2.19.1
pynndescent==0.5.13
pyopenjtalk==0.4.0
pyparsing==3.1.1
pypinyin==0.50.0
python-dateutil==2.8.2
python-mecab-ko==1.3.7
python-mecab-ko-dic==2.1.1.post2
python-multipart==0.0.20
pytorch-lightning==2.1.3
pytorch-wpe==0.0.1
pytz==2023.3.post1
PyYAML==6.0.1
referencing==0.32.1
regex==2023.12.25
requests==2.32.3
requests-oauthlib==1.3.1
resampy==0.4.2
rich==13.9.4
robust-downloader==0.0.2
rotary-embedding-torch==0.8.6
rpds-py==0.17.1
rsa==4.9
ruff==0.9.7
safetensors==0.5.2
scikit-learn==1.4.0
scipy==1.12.0
semantic-version==2.10.0
sentencepiece==0.1.99
setuptools==68.2.2
shellingham==1.5.4
simplejson==3.19.2
six==1.16.0
sniffio==1.3.0
sortedcontainers==2.4.0
soundfile==0.12.1
split-lang==2.1.0
starlette==0.35.1
sympy==1.12
tensorboard==2.15.1
tensorboard-data-server==0.7.2
tensorboardX==2.6.2.2
threadpoolctl==3.2.0
tiktoken==0.9.0
ToJyutping==3.2.0
tokenizers==0.21.0
tomli==2.0.1
tomlkit==0.12.0
toolz==0.12.0
torch==2.1.2
torch-complex==0.4.3
torchaudio==2.1.2
torchmetrics==1.3.0.post0
tqdm==4.66.1
transformers==4.49.0
triton==2.1.0
typer==0.15.1
typing_extensions==4.9.0
tzdata==2023.4
uc-micro-py==1.0.2
umap==0.1.1
umap-learn==0.5.7
urllib3==2.1.0
uvicorn==0.27.0
websockets==11.0.3
Werkzeug==3.0.1
wheel==0.41.2
wordsegment==1.3.1
x-transformers==2.1.2
xxhash==3.4.1
yapf==0.40.2
yarl==1.9.4
zipp==3.17.0
